loki/backend/cgen.py

```diff
@@ -140,6 +143,25 @@ def map_c_reference(self, expr, enclosing_prec, *args, **kwargs):
     def map_c_dereference(self, expr, enclosing_prec, *args, **kwargs):
         return self.format(' (*%s)', self.rec(expr.expression, PREC_NONE, *args, **kwargs))
 
+    def map_inline_call(self, expr, enclosing_prec, *args, **kwargs):
+
+        class FindFloatLiterals(ExpressionFinder):
```

Comment:
Just a minor nitpick, but I think this is general enough to live in `loki.ir.expr_visitor.py`